docs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.7-alpine
+FROM ruby:3.1-alpine
```

Comment:
@shwoodard do you have any idea if this would be an issue for the website? I'm honestly not familiar with site part of the stack.

Reply:
I'm not sure. Does one depend on the other or is this just local to this repo?

Reply:
Why could try it and revert it if it breaks or is there a way to test locally?